privacyidea/lib/policy.py

```diff
@@ -314,6 +314,7 @@ class ACTION(object):
     TOKENREALMS = "tokenrealms"
     TOKENTYPE = "tokentype"
     TOKENINFO = "tokeninfo"
+    TOKENINFO_API_EXCLUDE = "tokeninfo_api_exclude"
```

Comment:
Please rename it to

HIDE_TOKENINFO = "hide_tokeninfo"

We have a similar policy "hide_audit_columns".